Content.Server/Chat/Commands/SuicideCommand.cs

```diff
@@ -83,6 +84,13 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 return;
             }
 
+            //Checks to see if the player is dead.
+            if(_entities.TryGetComponent<MobStateComponent>(owner, out var mobState) && mobState.IsDead())
+            {
+                shell?.WriteLine("You can't suicide, you're dead.");
```

Comment:
Use localization here so that it can be translated. `Loc.GetString()` and the corresponding ftl entry.

Also if you're keeping this string then put a period instead of a comma.

Reply:
Would I still be putting it to the shell?  Also what using statement do I need for Loc.GetString?  I've forgotten and can't seem to find it.  I thought it was the Localization using, but it stays greyed out and won't pop up when I start typing it in.

Reply:
Nevermind about the above, for some reason it wasn't loading in properly on Rider and VS2022, but it did on VSC.